docs/topics/migrations.txt

```diff
@@ -218,6 +218,26 @@ Apps without migrations must not have relations (``ForeignKey``,
 ``ManyToManyField``, etc.) to apps with migrations. Sometimes it may work, but
 it's not supported.
 
+Swappable dependencies
+----------------------
+
+.. function:: django.db.migrations.swappable_dependency(value)
+
+The ``swappable_dependency()`` function is used in migrations to
+declare "swappable" dependencies on the first migration in the app of the
+swapped-in model. If the swapped-in model isn't created as part of the initial
+migration of an app, the dependencies can be wrong, causing migration
+failures. The argument ``value`` is a string ``"<app label>.<model>"``
```

Comment:
I'd remove _"If the swapped-in model isn't created as part of the initial migration of an app, the dependencies can be wrong, causing migration failures."_ or change it to e.g. _"Therefore, swapped-in model should be included in the initial migration."_.

We documented how it works, IMO, we don't need to scary users, and it may not be fixable, see ticket-23694.